src/kernel/vga.zig

```diff
@@ -0,0 +1,483 @@
+// Zig version: 0.4.0
+
+// ------------------------------------------------------------------------------------------------
+// Imports
+// ------------------------------------------------------------------------------------------------
+
+//const arch = @import("arch.zig");
+const builtin = @import("builtin");
+const arch = if (builtin.is_test) @import("arch_mock.zig")
```

Comment:
I would suggest that we put mock files under a separate directory, perhaps pluto/test, just to keep the source structure a bit cleaner and more lightweight. What do you think?

Reply:
I have put the mocked files in a test dir, but it does make the imports longer